src/coreclr/tests/issues.targets

```diff
@@ -986,4 +986,710 @@
         </ExcludeList>
     </ItemGroup>
 
+    <!-- Known failures on the mono runtime -->
+    <ItemGroup Condition="'$(RuntimeName)' == 'mono'" >
```

Comment:
What OS/Arch is this disabling tests for?

Reply:
Are these tests that are completely incompatible with the mono runtime?

Reply:
That's my understanding; maybe "completely incompatible" is a bit too strong statement, I guess that some of them may [and perhaps should] be fixed to be mono-friendly. Not sure about bits where the two runtimes fundamentally differ like GC.

Reply:
Correct, the question is really to understand what this when this new bucket is actually attempting to disable and why.

Reply:
For example, if these are just the known set of tests that fail on mono/x64/linux then I would like us to annotate it as such.

Reply:
I think that should be part of the triage the test claim to need, the important bit is to make sure this triage actually happens at some point. Perhaps the hypothetical issue cleanup bot could also emit warnings for tests without valid work item ID's so that they could be brought up at infra sync-ups and such.

Reply:
Many of these I believe will fail on Mono across all architectures, although I have chiefly been testing on OSX/x64. I am definitely aware that I am going to have to create some more categories, but it is a process.

I don't object to adding a RunTime == CoreCLR check to the existing exclusions, but I will have to figure out which of those also fail on Mono and which do not.

As for creating the actual issues, I plan to create them this week.